pkg/logger/logger.go

```diff
@@ -6,199 +6,122 @@ package logger
 import (
 	"flag"
 	"fmt"
-	"io"
-	"log"
-	"os"
 	"strings"
-)
-
-type LogLevel int
 
-const (
-	LevelFatal LogLevel = iota
-	LevelError
-	LevelWarning
-	LevelInfo
-	LevelDebug
-	LevelTrace
+	"go.uber.org/zap"
 )
 
 var (
 	quiet       = flag.Bool("quiet", false, "Disables all logging output")
 	logLevelArg = flag.String("log-level", "info", "Sets the maximum level of output [Fatal, Error, Warning, Info (Default), Debug, Trace]")
 	logFile     = flag.String("log-file", "", "")
+	logFormat   = flag.String("log-format", "juice", "Set the format of the logging [juice, console, json]")
```

Comment:
What is the difference between juice and console?

Reply:
Console is zap's development display - probably should just gate it to juice or json. The console one is pretty nice but doesn't display a single line for anything > INFO or something as it'll display a call stack in those situations